deis/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
```

Comment:
Yes you'll need to enable the daemonsets api. I'll make sure the docs get
updated.
On Dec 15, 2015 12:03 PM, "Matthew Fisher" <notifications@github.com> wrote:

> In deis/manifests/deis-logger-fluentd-daemon.yaml
> <https://github.com/deis/charts/pull/27#discussion_r47681267>:
>
> > @@ -0,0 +1,41 @@
> > +apiVersion: extensions/v1beta1
>
> do we need to install any extra extensions in order for daemonsets to work
> in k8s v1.1 or is this ready OOTB? Just wondering if there needs to be
> additions to the installation docs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deis/charts/pull/27/files#r47681267>.
>